pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -195,6 +197,11 @@ func (ci *CloudInfo) collectInfo(ic *types.InstallConfig, opts *clientconfig.Cli
 		return err
 	}
 
+	ci.VolumeTypes, err = ci.getVolumeTypes()
```

Comment:
Possibly, we only gather the volume types when using root volumes.

Reply:
ack I'll do it later